docs/section16/Rule16-1.md

```diff
@@ -0,0 +1,53 @@
+# Elevators – Rule 16-1  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 16-1  
+**Rule Description:** The elevator peak motor power shall be calculated according to the equation in Table G3.1-16  
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-16  
+**Data Lookup:** Table G3.9.1, Table G3.9.2, Table G3.9.3  
+**Evaluation Context:** Each elevator  
+
+**Applicability Checks:**  
+  1. The proposed building has an elevator.  
+
+**Function Call:**  
+N/A
+
+**Applicability Check 1:**
+- Rule is applicable if any building in the proposed RMD has an elevator: `if any(len(building.elevators) > 0 for building in P_RMD...buildings:`
+
+## Rule Logic:
+- For each building in the baseline RMD: `for building in B_RMD...buildings`
+  - Get the number of floors above grade in the building: `floors_above_grade_b = building.number_of_floors_above_grade`
+  - Get the number of floors below grade in the building: `floors_below_grade_b = building.number_of_floors_above_grade`
+  - If both floors above grade and floors below grade are null: `if floors_above_grade_b == Null and floors_below_grade_b == Null: is_undetermined = true`
+  - Elif floors above grade is null: `elif floors_above_grade_b == Null: total_floors_b = floors_below_grade_b`
+  - Elif floors below grade is null: `elif floors_below_grade_b == Null: total_floors_b = floors_above_grade_b`
+  - Else, calculate the total number of floors in the building: `else: total_floors_b = floors_above_grade_b + floors_below_grade_b`
```

Comment:
I think we need a number_of_floors_served data element added to the Elevator data table. Each elevator does not neccesarily serve all floors in the building.

Reply:
Agreed. Otherwise, if number of floors is being calculated here, it should either use the existing function get_number_of_floors, or follow the same logic for consistency with section 18.